### PR TITLE
[Docs] pg_stat_activity view #6519

### PR DESCRIPTION
**Pull request by ddhodge**
pg_stat_activity view #6519

### CODE REVIEW

File: docs/content/latest/develop/query/pg_stat_activity.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+The `pg_stat_activity` view displays one row per server process, showing information related to the current activity of that process.
+
+The fields and their values are described in the following table.
+
+Field | Type | Description
+--- | --- | ---
+datid | oid | The object identifier (OID) of the database to which the backend is connected.
+datname | name | The name of the database to which the backend is connected.
```
Comment (polarweasel) [inline docs/content/latest/develop/query/pg_stat_activity.md:30]: Style preferences: markdown tables should be "outlined" with pipes. Also, be sure to specify the alignment for each column (my suggestion left-aligns them all).

There's no need to make the heading rows wide enough so the table actually formats as a table in your text editor, like some people do, but each row should be obviously a table row, from beginning to end.

```suggestion
| Field | Type | Description |
| :---- | :--- | :---------- |
| datid | oid | The object identifier (OID) of the database to which the backend is connected. |
| datname | name | The name of the database to which the backend is connected. |
```

**Comment by polarweasel** [comment]
Hey @ddhodge I've taken a couple of your changes and added them to the other PR for this issue. After you've looked at my comment, you can close this PR...

File: docs/content/latest/develop/query/pg_stat_activity.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+pid | integer | The ID of the backend process.
+usesysid | oid | The OID of the user.
+usename | name | The name of the user.
+application_name | text | The name of the application that is connected to this backend.
+client_addr | inet | The IP address of the client connected to this backend. If this field is empty, the client is connected through a Unix socket on the server or this is an internal process such as autovacuum.
+client_hostname | text | The host name of the client connected to this backend, as reported by a reverse DNS lookup of client_addr.
+client_port | integer | The TCP port number that the client uses for communication with the backend server. A value of -1 indicates that a Unix socket is used.
+backend_start | timestampz | Time when the current backend process was started.
+xact_start | timestampz | The time when the current transaction of this process was started, or null if no transaction is active. If the current query is the first transaction of the process, this field is equivalent to the query_start field.
```
Comment (lizayugabyte) [inline docs/content/latest/develop/query/pg_stat_activity.md:39]: Suggestion: 
The time > Time

**Review by lizayugabyte — COMMENTED** [review]
One minor comment. Otherwise LGTM

**Comment by lizayugabyte** [comment]
@ddhodge You might also want to close https://github.com/yugabyte/yugabyte-db/issues/6519 as "done".

**Comment by polarweasel** [comment]
Please don't close 6519. There's another PR open that will do that

**Pull request closed without merging**